inc/core/patterns/bcgov-alternating-cards.php

```diff
@@ -12,7 +12,8 @@
 return array(
 	'title'      => __( 'Alternating Cards', 'bcgov_blocks_theme' ),
 	'categories' => array( 'bcgov-blocks-theme-general' ),
-	'content'    => '<!-- wp:media-text {"mediaPosition":"right","mediaId":305,"mediaLink":"' . esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/default_10.jpg","mediaType":"image","verticalAlignment":"center","style":{"color":{"background":"#0033661a"}}} -->
+	'content'    => '<!-- wp:group {"align":"wide","style":{"spacing":{"margin":{"top":"3rem","bottom":"3rem"}}}} -->
+	<div class="wp-block-group alignwide" style="margin-top:3rem;margin-bottom:3rem"><!-- wp:media-text {"mediaPosition":"right","mediaId":305,"mediaLink":"' . esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/default_10.jpg","mediaType":"image","verticalAlignment":"center","style":{"color":{"background":"#0033661a"}}} -->
```

Comment:
It's hard to review block templates when they're just one string, crammed with CSS, comments, various layers of HTML. If we keep doing block pattern themes going forward I feel like we may need to reach for tools or techniques that make this easier to parse. Not a blocker for now of course